### PR TITLE
Vox goodies update!

### PR DESCRIPTION
**Pull request by crabcube**
Tiny and my first effort at **CONTENT**
God I hope I am doing this right and don't destroy everything when I push this button.

Adds an engineering locked EVA/CE hardsuit hybrid to cargo for vox. 
Updates insulated gauntlets with heavy duty engineering gloves armour values
Updates magtalons with similar to work boots armour values

### CODE REVIEW

**Comment by Darlantanis** [comment]
This does not compile. 
image

**Comment by crabcube** [comment]
I have done my very best

**Comment by Razgriz1032** [comment]
1. Your description lies because you're adding armor values, not just "Just adds an engineering locked EVA/CE hardsuit hybrid to cargo for vox."
image

2. No need to add values to </obj/item/clothing/shoes/magboots/vox> because of inheritance from </obj/item/clothing/shoes/magboots>

3. That armor for magboots will crazy stack with whatever else the character is wearing, so anyone who puts on a hardsuit or voidsuit will have extra tanky values.

File: code/modules/clothing/spacesuits/rig/suits/alien.dm [hunk 1]
```diff
@@ -152,3 +152,39 @@
 
 	initial_modules = list(
 		)
+//ChompEdit Begins
+
+/obj/item/weapon/rig/vox/engineering
```
Comment (Razgriz1032) [inline code/modules/clothing/spacesuits/rig/suits/alien.dm:157]: You are missing `req_access = list(access_engine)` if you want to have this locked to engineering only

File: code/modules/clothing/spacesuits/rig/suits/alien.dm [hunk 2]
```diff
@@ -152,3 +152,39 @@
...
+	offline_vision_restriction = 1
+
+	initial_modules = list(
+	)
+
+	air_type = /obj/item/weapon/tank/vox
+
+	max_heat_protection_temperature = FIRE_HELMET_MAX_HEAT_PROTECTION_TEMPERATURE
+
+/obj/item/weapon/rig/vox_engine_technition_control_module/equipped
```
Comment (Razgriz1032) [inline code/modules/clothing/spacesuits/rig/suits/alien.dm:180]: This inherits nothing from `/obj/item/weapon/rig/vox/engineering` but instead inherits all from `/obj/item/weapon/rig/vox`
https://github.com/CHOMPStation2/CHOMPStation2/blob/30843dfb81eb5a059ae81c91cda839488ad18147/code/modules/clothing/spacesuits/rig/suits/alien.dm#L57

**Comment by crabcube** [comment]
Finally functions, merge success and tested. 
What I should have done the first time.

**Pull request merged**